packages/react-wildcat/src/server.js

```diff
@@ -45,11 +45,18 @@ function start() {
     const __TEST__ = (process.env.BABEL_ENV === "test");
     const __DEBUG__ = (process.env.DEBUG);
 
-    let cpuCount = os.cpus().length;
+    let cpuCount = (appServerSettings && appServerSettings.maxClusterCpuCount) || Infinity;
+
+    if (cpuCount === Infinity) {
+        cpuCount = os.cpus().length;
+    }
 
     /* istanbul ignore else */
     if (!__PROD__ || __TEST__) {
         https.globalAgent.options.rejectUnauthorized = false;
+    }
+
+    if (__TEST__ && !wildcatConfig.__ClusterServerTest__) {
```

Comment:
I eventually want to have a `testMode` property in `wildcat.config.js` rather than relying on an env var. It's tricky though since the test runners use env vars to enable that.